src/SerializeResult.h

```diff
@@ -0,0 +1,27 @@
+////
+//  SerializeSourceAnnotations.h
+//  drafter
+////
```

Comment:
I think you mistyped this line. Shouldn't it be `//`? Similarly the 1st line of this file.